recipes/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set data = load_setup_py_data() %}
+{% set version = data.get('version') %}
+
+package:
+  name: "openghg_defs"
+  version: {{ version }}
```

Comment:
When I ran this, the version came out as `None` so the `load_setup_py_data` is probably not working in some way.

Reply:
That's been replaced now.